app/src/main/res/values/strings.xml

```diff
@@ -25,4 +25,95 @@
     <string name="event_note">Note</string>
     <string name="drawer_open">open</string>
     <string name="drawer_close">close</string>
+    <string name="large_text">
```

Comment:
I guess that large_text was used for testing, do we still need it?

Reply:
Removed.